jsf/subsystem/src/test/java/org/jboss/as/jsf/deployment/JSFModuleIdFactoryTestCase.java

```diff
@@ -52,25 +54,44 @@ public void validSlotTest() {
     @Test
     @Ignore("Depends on https://issues.redhat.com/browse/WFLY-17405")
```

Comment:
@RanabirChakraborty The change here 'works' only because of ^^^. If this is removed, the test fails. OTOH the test would pass on 'main' (i.e. on main this @Ignore can/should be removed, since WFLY-17405 is resolved and this passes.)

Please do remove this once you get the test passing. :)